### ISSUE TITLE
Screenreader: some of the operators like (  ) - and +/- are missing text

### ISSUE THREAD

**Issue by oucnelson** [RUB-ITS-SL-Moodle/moodle-block_calculator#14]
Assuming this video attaches, it demonstrates that there needs to be some aria tags adding to the calculator, as some of the operators like (  ) - and +/- are missing text.

https://github.com/user-attachments/assets/849a4fda-1a94-4c9f-9032-3419ee105fc9
